demo/index.md

```diff
@@ -3,34 +3,30 @@ layout: default
 markdown: kramdown
 permalink: demo/
 ---
-# Angry Bots Demo
-<div markdown="1" id="wasm-fail" class="flash flash-error flash-hide">
-  Uh-oh! WebAssembly isn't enabled in this browser. To get an early preview of this experimental technology, at your own risk:
+# Tanks! Demo
 
-  - on [Chrome Canary](https://www.google.com/chrome/browser/canary.html), open `chrome://flags/#enable-webassembly` and enable the switch.
-  - on [Firefox Nightly](https://nightly.mozilla.org/), open `about:config` and set `javascript.options.wasm` to `true`.
-
-  See a [preview](http://blogs.windows.com/msedgedev/2016/03/15/previewing-webassembly-experiments) of [Microsoft Edge](https://www.microsoft.com/en-us/windows/microsoft-edge) support and follow [Safari](http://www.apple.com/safari/) support on WebKit's [feature status](https://webkit.org/status/#specification-webassembly) page.
+[![](screenshot.jpg)](Tanks/)
+<div id="play-wasm" class="btn-block">
+  <a class="btn btn-primary" href="Tanks/" role="button">Play WebAssembly</a>
+</div>
+<div id="play-asm" class="btn-block hide-btn-block">
+  <a class="btn hide-asm-support" href="Tanks/" role="button">Play asm.js fallback</a>
+  <span class="btn-comment btn-comment-error hide-asm-support">Your browser doesn't support WebAssembly yet. <a href="/roadmap/">Learn more</a></span>
 </div>
 
-[![](screenshot.jpg)](AngryBots/)
-<a class="btn btn-primary" id="play-wasm" href="AngryBots/" role="button">Play WebAssembly</a>
-<a class="btn" href="http://beta.unity3d.com/jonas/AngryBots/" role="button">Play asm.js fallback</a>
-
-This is an experimental demo of Angry Bots, a Unity game which has been ported to WebAssembly. Fight robots and explore a 3D space station with realistic environmental effects in this top-down shooter. Movement is controlled by W, A, S, D or the arrow keys and aiming &amp; firing is controlled by the mouse.
+This is a demo of Tanks!, a Unity tutorial game which has been ported to WebAssembly. Drive a tank around the sandbox and shoot the enemy tank. Movement is controlled by W, A, S, D keys and firing is controlled by the spacebar.
```

Comment:
Could you link "Tanks! a Unity tutorial game" to https://unity3d.com/learn/tutorials/projects/tanks-tutorial?  And then might be good to expand "ported to WebAssembly" to "automatically ported to WebAssembly using the experimental WebGL export"